src/controllers/schemas.ts

```diff
@@ -0,0 +1,145 @@
+export const paginationSchema = {
+  querystring: {
+    type: "object",
+    properties: {
+      page: {
+        type: "number",
+        default: 1,
+      },
+      limit: {
+        type: "number",
+        default: 100,
+      },
+      status: {
+        type: "string",
+        enum: ["pending", "executed", "failed"],
+      },
+    },
+  },
+}
+
+export const senderSchema = {
+  querystring: {
+    type: "object",
+    properties: {
```

Comment:
Extract the pagination properties and status properties to a separate schema and change the limit default to 100.

e.g. 
```
 const paginationSchema = {
   page: {}
   limit: {}
}

const transferStatusSchema = {
    status: {
      type: string
      enum: ....
   }
}

 const senderSchema = {
   queryString: {
     type: "object",
     properties: {
        ...paginationSchema,
        ...statusSchema
    }
}
}
```

Do that for all schemas.